api/src/controllers/transferCredits.ts

```diff
@@ -120,7 +100,7 @@ const transferCreditsRouter = router({
       .where(eq(transferredGe.userId, ctx.session.userId!));
     return response as TransferredGE[];
   }),
-  setTransferredGE: userProcedure.input(z.object({ GE: zodGETransferSchema })).mutation(async ({ input, ctx }) => {
+  setTransferredGE: userProcedure.input(z.object({ GE: zodTransferredGE })).mutation(async ({ input, ctx }) => {
```

Comment:
if you're updating the types, it'd probably be good to update this and make it more consistent as well.

```suggestion
  setTransferredGE: userProcedure.input(zodTransferredGE)).mutation(async ({ input, ctx }) => {
```

Reply:
Done